deploy/templates/auth/principals/service-clients.yaml

```diff
@@ -122,3 +122,47 @@ spec:
       group: {{ .Values.acs.organisation }}-Service-Core
       node: Monitor
 {{- end }}
+---
+{{- if .Values.unsIngesters.sparkplug.enabled }}
+apiVersion: factoryplus.app.amrc.co.uk/v1
+kind: KerberosKey
+metadata:
+  name: sv1sparkplugingester
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: Random
+  principal: sv1sparkplugingester@{{ .Values.identity.realm | required "values.identity.realm is required!" }}
+  secret: uns-ingester-sparkplug-keytabs/client
+  account:
+    class: e463b4ae-a322-46cc-8976-4ba76838e908
+    name: Sparkplug Ingester
+    groups:
+      - e414d355-b991-429b-8f5d-97e823ff71f5
+    sparkplug:
+      group: {{ .Values.acs.organisation }}-Service-Core
+      node: SparkplugIngester
+
+{{- end }}
+---
+# In the future if we have any more ingesters then this should be
+# enabled if any of the ingesters are enabled.
+{{- if .Values.unsIngesters.sparkplug.enabled }}
```

Comment:
You already have `historians.uns`. The UNS republishing can be useful as an alternative to needing to read Sparkplug for live data without duplicating the data in the historian.

Reply:
I agree, I've changed it to use the historian uns value.